server/src/main/java/io/crate/planner/operators/ProjectSet.java

```diff
@@ -150,7 +149,7 @@ public List<Symbol> standaloneOutputs() {
 
     @Override
     public LogicalPlan pruneOutputsExcept(Collection<Symbol> outputsToKeep) {
-        HashSet<Symbol> toKeep = new HashSet<>();
+        HashSet<Symbol> toKeep = new LinkedHashSet<>();
```

Comment:
Looks like this is only used for containment check and not as output. I think this can be kept as HashMap?